protocol/avdl/keybase1/teams.avdl

```diff
@@ -208,6 +208,11 @@ protocol teams {
   @typedef("string")
   record UserVersionPercentForm {}
```

Comment:
I would take this opportunity to add a comment here:
```
// A (username, seqno) pair.
// The username is adorned with "%n" at the end
// where n is the seqno IF the seqno is not 1.
```
I took this from `chain_parse.go:26`, but I think there the comment is attached to the wrong definition.